packages/venia-concept/src/components/MiniCart/mask.js

```diff
@@ -1,33 +1,33 @@
-import React, { Component } from 'react';
+import React from 'react';
 import { bool, func, shape, string } from 'prop-types';
 
+import { mergeClasses } from 'src/classify';
 import Mask from 'src/components/Mask';
 
-import classify from 'src/classify';
 import defaultClasses from './mask.css';
 
-class MiniCartMask extends Component {
-    static propTypes = {
-        classes: shape({
-            root_active: string
-        }),
-        dismiss: func,
-        isActive: bool
-    };
+const MiniCartMask = props => {
```

Comment:
This `Mask` passes a different `opacity` to the `Mask` in `src/components` for partial blocking of the MiniCart while the checkout flow is open:

![Screen Shot 2019-05-31 at 4 38 02 PM](https://user-images.githubusercontent.com/13182778/58736167-7b7a0880-83c2-11e9-8e50-48aef6b7001b.png)

But you did make me realize that I had a regression here because I missed adding this component back to `MiniCart` 😬 😮 . Resolved by https://github.com/magento-research/pwa-studio/pull/1266/commits/7eea1855e56030b6e8cb56f07ec1a79e1e016849.